### PR TITLE
Improve designs, icons, and fix several issues

### PR DESCRIPTION
**Pull request by cathirv**
This PR is pending another (https://github.com/Kommunicate-io/Kommunicate-Android-Chat-SDK/pull/5), so the first commit would be --> Refactor icons -->933220e

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,22 +1,11 @@
...
-
     repositories {
         google()
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.1'
-        classpath 'com.google.gms:google-services:3.1.1'
-
-       /* classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.1'
```
Comment (reytum) [inline build.gradle]: These 4 lines are needed for releasing the library

File: kommunicate/build.gradle [hunk 2]
```diff
@@ -21,75 +18,10 @@ android {
...
-
 }
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    api project(':kommunicateui')
     implementation 'com.android.support:appcompat-v7:27.1.1'
+    api project(':kommunicateui')
 }
-
```
Comment (reytum) [inline kommunicate/build.gradle]: These lines are needed to release the library

File: kommunicateui/build.gradle [hunk 3]
```diff
@@ -23,43 +23,15 @@ dependencies {
...
+    api 'com.google.android.gms:play-services-maps:15.0.1'
+    api 'com.google.android.gms:play-services-location:15.0.1'
+    api 'com.android.support:design:27.1.1'
     api 'com.rockerhieu.emojicon:library:1.3.3'
-    implementation 'com.android.support.constraint:constraint-layout:1.1.1'
-    //compile project(':mobicomkit')//Note: use this for customization
     api 'com.applozic.communication.message:mobicomkit:5.8.7'
-    //Note: use this in case customization is not required
-}
-/*ext {
```
Comment (reytum) [inline kommunicateui/build.gradle]: These lines are needed to release the library

**Comment by reytum** [comment]
@cathirv Can you please split the PRs into 2 different PRs. One with only improvements that doesn't changes the design completely(Except for material designs) and the other one with your new designs like the message sender view(bottom view) and the message status icons(read, delivered, sent). This will help us in direclty mergin the PRs with improvements and cherry picking the one with designs.
Thanks for your contributions though 👍

**Comment by cathirv** [comment]
@reytum I already have the two PRs divided. But they are independent to this. So you could close it.

**Comment by cathirv** [comment]
I close PR because I have divided it into next three:
https://github.com/Kommunicate-io/Kommunicate-Android-Chat-SDK/pull/7, https://github.com/Kommunicate-io/Kommunicate-Android-Chat-SDK/pull/8 y https://github.com/Kommunicate-io/Kommunicate-Android-Chat-SDK/pull/9

**Pull request closed without merging**